.github/workflows/preview-build.yml

```diff
@@ -112,6 +117,7 @@ jobs:
         id: check-files
         uses: tj-actions/changed-files@2f7c5bfce28377bc069a65ba478de0a74aa0ca32 # v46.0.1
         with:
+          safe_output: false # set to false because we are using an environment variable to store the output and avoid command injection.
```

Comment:
How does this work? won't this still emit unsanitzed files names on standard out? 

Is setting this unsafe because of the action or because of possible usages of the `steps.check-files.outputs.all_changed_files` in follow up scripts?

Reply:
I got this from the [examples section](https://github.com/tj-actions/changed-files?tab=readme-ov-file#examples-).

The "Get all changed files without escaping unsafe filename characters" example. 

But I might have copy-pasted to soon. I guess this is not needed. Thanks for noticing!

Reply:
https://github.com/elastic/docs-builder/pull/1416/commits/6c50add5b2affa7ccb375aaf7306e78250c3f360